Classes/EventListener/ModifyFileDumpEventListener.php

```diff
@@ -205,14 +170,13 @@ public function checkFileAccess(ResourceInterface $file)
     /**
      * Dump file contents
      *
-     * @todo: find a nicer way to force the download. Other hooks are blocked by this.
-     * @todo: Try to get the resumable option part of TYPO3 core itself
+     * TODO: Try to get the resumable option part of TYPO3 core itself find a nicer way to force the download. Other hooks are blocked by this.
      *
      * @param File $file
      * @param bool $asDownload
      * @param bool $resumableDownload
      */
-    protected function dumpFileContents($file, $asDownload, $resumableDownload)
+    protected function dumpFileContents(File $file, bool $asDownload, bool $resumableDownload)
```

Comment:
`$file` can also be `ProcessedFile` (this was already wrong in the annotation, but that didn't cause any problems). `File` can be replaced with `FileInterface`